app/code/Magento/CatalogWidget/Test/Mftf/Test/StorefrontAddProductToCartThroughAjaxTest.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="StorefrontAddProductToCartThroughAjaxTest">
+        <annotations>
+            <features value="Product List Widget"/>
+            <stories value="Add product to cart through ajax in Product List widget"/>
+            <title value="Add product through ajax"/>
+            <description value="Add product to cart through ajax in Product List widget on Cms page"/>
+        </annotations>
+        <before>
+            <!-- 1. Create multiple products and assign to a category. -->
```

Comment:
Please, remove comments from the test. They will appear as separate steps in test reports